test/albums.test.js

```diff
@@ -14,6 +14,21 @@ const mockedUser = {
   password: 'Wolox1189!'
 };
 
+const otherUser = {
```

Comment:
you already have many mocks, think in put them in a file like `util.js` for example